docs/source/installation/ray.rst

```diff
@@ -44,66 +45,73 @@ Creating a Mars on Ray runtime in the Ray cluster and do the computing:
     # Convert mars dataframe to ray dataset
     ds = md.to_ray_dataset(df)
     print(ds.schema(), ds.count())
-    ds.filter(lambda row: row["a"] > 0.5).show(5)
+    ds.filter(lambda row: row['a'] > 0.5).show(5)
     # Convert ray dataset to mars dataframe
     df2 = md.read_ray_dataset(ds)
     print(df2.head(5).execute())
 
 
-Create a Mars on Ray runtime independently in the Ray cluster:
+Start a Ray actor for Mars supervisor:
 
 .. code-block:: python
 
     import mars
-    import mars.tensor as mt
-    cluster = mars.new_cluster_in_ray(worker_num=2, worker_mem=2 * 1024 ** 3)
+    # Start a Ray actor for Mars supervisor.
+    session = mars.new_ray_session(backend='ray')
```

Comment:
There are `mars.new_session(backend='ray')` and `mars.new_ray_session(backend='ray')` here which may confuse users. Maybe we should specifically explain their differences.

Reply:
Good suggestion. Let me refine the doc.